src/styles/settings.plugin.scss

```diff
@@ -42,7 +42,8 @@
 			cursor: pointer;
 			font-size: 12px;
 			letter-spacing: .01rem;
-			padding: 4px 0 6px;
+			padding: 4px 0 6px 18px;
```

Comment:
why left padding changed?

Reply:
to align items with 'Back' text as showed https://app.zeplin.io/project/570f637c9e464b033a0bdef8/screen/59dea97aec084f180736458c

Reply:
Did it pass design review?

Reply:
old 
<img width="132" alt="screen shot 2017-11-13 at 12 32 12" src="https://user-images.githubusercontent.com/6940495/32724160-847de19a-c870-11e7-82d3-567e6746061b.png">
vs new
<img width="152" alt="screen shot 2017-11-13 at 12 31 37" src="https://user-images.githubusercontent.com/6940495/32724162-85e9099c-c870-11e7-8fd3-02fa1ae20086.png">

@hakubo, does it look acceptable? 🙈

Reply:
lgtm